msteams-platform/concepts/design/planning-checklist.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Questions to help plan Teams app development
 author: heath-hamilton
-description: Questions to consider while you plan your app, understand your user and their need, understand the user problems that your app would solve, plan user authentication and their onboarding experience
+description: Questions to consider while you plan your app, understand your user and their needs, problems that your app solves, user authentication and their onboarding experience.
```

Comment:
Doesn't read well. Needs edit. This is how I read it.

```
Questions to consider
 while you plan your app,
 understand your users and their needs,
 problems that your app solves,
 user authentication and their onboarding experience.
```

Reply:
Addressed in this PR: https://github.com/MicrosoftDocs/msteams-docs/pull/6385